spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/ReadIntegrationTestBase.java

```diff
@@ -47,6 +51,12 @@ private Dataset<Row> readFromTable(String table) {
         .load();
   }
 
+  @Test
+  public void testDatasetFilters_count() {
```

Comment:
testDataset_count()  ?

Reply:
Yeah, that's a better testname indeed, thanks!